addons/crm_survey/tests/test_crm_survey.py

```diff
@@ -0,0 +1,257 @@
+from odoo.addons.survey.tests import common
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+from odoo.addons.mail.tests.common import MockEmail
+
+
+@tagged('-at_install', 'post_install', 'functional')
+class TestCrmSurvey(common.TestSurveyCommon, MockEmail, HttpCase):
+    """
+    These tests will check:
+    - 1st case: if connected user's inputs contains "Create lead" answers, then a lead is created successfully
+    - 2nd case: if connected user's inputs not contain "Create lead" answers, a lead isn't created anymore
+    - 3rd case: if not connected user's inputs contains "Create lead" answers, then a lead is created with his email answer
+    """
+    def _create_lead_qualification_survey(self, is_in_sales_team=False, survey_name=None, survey_type="survey"):
+        login = "survey_manager"
+        # Adding in a sales team
+        if is_in_sales_team:
+            sales_team = self.env['crm.team'].create({
+                'name': 'Odoo Survey Team',
+                'use_leads': True
+            })
+            user = self.env['res.users'].search([('login', '=', login)])
```

Comment:
Why not directly using self.survey_manager? You inherit from TestSurveyCommon, so you should be able to use it right?